src/helpers/wagmiConfig.ts

```diff
@@ -0,0 +1,41 @@
+import { createConfig, mainnet } from 'use-wagmi';
+import { createPublicClient, http } from 'viem';
+import { InjectedConnector } from 'use-wagmi/connectors/injected';
+import { CoinbaseWalletConnector } from 'use-wagmi/connectors/coinbaseWallet';
+import { WalletConnectConnector } from 'use-wagmi/connectors/walletConnect';
+import { SafeConnector } from 'use-wagmi/connectors/safe';
+
+const config = createConfig({
+  // TODO: Enable after onConnect is fixed https://github.com/unicape/use-wagmi/issues/82
+  // autoConnect: true,
```

Comment:
I've disabled autoConnect (reconnect) until the issues from the comment is fixed